ag-grid-webcomponent/index.js

```diff
@@ -29,7 +30,7 @@ class AgGrid extends HTMLElement {
   }
 
   loadAttribute(attr) {
-    let gridElement = document.querySelector("ag-grid").getAttribute(attr);
+    let gridElement = this.getAttribute(attr);
```

Comment:
Would have been a bit nicer to have this in a separate commit because it is not really related to the renderer stuff. But I am fine with keeping it like this.